apps/web/components/booking/pages/BookingPage.tsx

```diff
@@ -860,7 +860,7 @@ const BookingPage = ({
                   {!eventType.disableGuests && !guestToggle && (
                     <Button
                       type="button"
-                      color="minimalSecondary"
+                      color="minimal"
```

Comment:
The `minimalSecondary` button doesn't exist anymore. I've converted all of them (only a few) to `minimal`.